Ginger/GingerCoreNET/RunLib/CLILib/CLIConfigFile.cs

```diff
@@ -142,34 +142,53 @@ public void LoadContent(string content, CLIHelper cliHelper, RunsetExecutor runs
-                        case "Env":
-                        case "env":
-                            cliHelper.Env = value;                        
-                            break;
-                        case "RunSet":
-                        case "runset":
-                            cliHelper.Runset = value;                            
-                            break;
+                            break;                                              
                         case "ShowAutoRunWindow": // Support old style
                         case "showui": // TODO: use CLIOptionClassHelper.GetAttrLongName<RunOptions>(nameof(RunOptions.ShowUI)):
                             cliHelper.ShowAutoRunWindow = bool.Parse(value);
-                            break;
-                        case "RunAnalyzer":
-                        case "analyze":
-                            cliHelper.RunAnalyzer = bool.Parse(value);                            
-                            break;
+                            break;                       
                         case "artifacts-path":
                             cliHelper.TestArtifactsFolder = value;
                             break;
-                        default:
-                            Reporter.ToLog(eLogLevel.ERROR, "Unknown argument: '" + param + "'");
-                            throw new ArgumentException("Unknown argument", param);
+                        //default:
+                        //    Reporter.ToLog(eLogLevel.ERROR, "Unknown argument: '" + param + "'");
+                        //    throw new ArgumentException("Unknown argument", param);
                     }
                 }                
             }            
         }
 
+        public void LoadRunsetConfigurations(string content, CLIHelper cliHelper, RunsetExecutor runsetExecutor)
+        {
+            using (System.IO.StringReader reader = new System.IO.StringReader(content))
+            {
+                string arg;
+                while ((arg = reader.ReadLine()) != null)
+                {
+                    int i = arg.IndexOf('=');
+                    string param = arg.Substring(0, i).Trim();
+                    string value = arg.Substring(i + 1).Trim();
 
+                    switch (param)
+                    {
+                        case "RunSet":
+                        case "runset":
+                            cliHelper.Runset = value;
+                            break;
+                        case "Env":
+                        case "env":
+                            cliHelper.Env = value;
+                            break;
+                        case "RunAnalyzer":
+                        case "analyze":
+                            cliHelper.RunAnalyzer = bool.Parse(value);
+                            break;
+                        //default:
+                        //    Reporter.ToLog(eLogLevel.ERROR, "Unknown argument: '" + param + "'");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)